src/http/routes.cc

```diff
@@ -53,7 +53,7 @@ routes::~routes() {
             delete r.second;
         }
```

Comment:
> Prevent leaking _default_handler

This repository requires patches that have an explanation, and requires the subject line to point to the subsystem that is modified.

Reply:
Done, hope now it is good to go